components/ShareWidget.vue

```diff
@@ -0,0 +1,174 @@
+<template>
+  <div
+    id="ShareWidget"
```

Comment:
id is normally reserved for one unique item and it takes priority in CSS. Preferably we would use class over id. That way the component can become reusable. Your css is targeting the class ShareWidget anyways, so I would remove the id. Unless you're using it in JS somewhere that I've overlooked?

Reply:
No, sorry this should have been removed and was there because of an earlier misunderstanding I had about how Vue exports. I will remove it.